src/libraries/System.Private.DataContractSerialization/src/System/Runtime/Serialization/XmlObjectSerializerReadContext.cs

```diff
@@ -381,23 +393,25 @@ internal virtual int GetArraySize()
 #if USE_REFEMIT
         public void AddNewObject(object obj)
 #else
-        internal void AddNewObject(object obj)
+        internal void AddNewObject(object? obj)
```

Comment:
is USE_REFEMIT path used anywhere? should both paths be annotated?

Reply:
It isn't defined anywhere. https://github.com/dotnet/runtime/search?q=USE_REFEMIT

There is also #20131, which says to remove it as it is not used. But that issue was closed without it being removed.

I can follow up in 6.0 to remove this unused define.

Reply:
I've opened https://github.com/dotnet/runtime/issues/41512 to remove this define.